### PR TITLE
CNS-6275 Add prismaRegion, projectID, and tags attributes to gcp resources

### PR DESCRIPTION
**Pull request by kcarrasco-panw**
Adds the attributes mentioned in the title to the gcpresource spec.
It also renames `prismaCloudRRN` to `prismaRRN` for naming consistency.

### CODE REVIEW

File: specs/gcpresource.spec [hunk 1]
```diff
@@ -71,8 +74,24 @@ attributes:
...
-  - name: prismaCloudRRN
-    description: The identifier used by Prisma Cloud to locate the same resource.
+  - name: prismaRRN
+    description: The resource identifier in PrismaCloud.
+    type: string
+    exposed: true
+    stored: true
+    omit_empty: true
+
+  - name: prismaRegion
```
[the remark this answers is not in the document]
Comment (kcarrasco-panw) [reply specs/gcpresource.spec:84]: Should I remove `prismaRegion` then?

**Comment by kcarrasco-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "neocnawrapper-gcp-support",
    "component": "backend",
    "pr-id": "1959",
    "commit-sha": "cc988abc5581ce902b5a53650c8eaf3549c16341",
    "pipeline": "master",
    "org": "prismacloud/pcn"
  },
  {
    "project": "neocnawrapper-gcp-support",
    "component": "gaia",
    "pr-id": "855",
    "commit-sha": "3842989b4d297be5216c13a16e4c3f1a3834d6b9",
    "pipeline": "master",
    "org": "aporeto-inc"
  }
]
```

**Comment by kcarrasco-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "neocnawrapper-gcp-support",
    "component": "backend",
    "pr-id": "1959",
    "commit-sha": "0820e6ee4e812bbe2256fd8a0a2e9d82f26c62e6",
    "pipeline": "master",
    "org": "prismacloud/pcn"
  },
  {
    "project": "neocnawrapper-gcp-support",
    "component": "gaia",
    "pr-id": "855",
    "commit-sha": "3842989b4d297be5216c13a16e4c3f1a3834d6b9",
    "pipeline": "master",
    "org": "aporeto-inc"
  }
]
```

**Review by malkhamis-panw — APPROVED** [review]
when you guys later touch the spec file again, let's sync flags like `auto_generated` and `read_only` etc like in azureresource.spec.

**Pull request merged**